client/app/components/form/FormBase.tsx

```diff
@@ -27,8 +42,9 @@ const FormBase: React.FC<FormPropsWithTheme<any>> = (props) => {
       formData={formData ?? {}}
       noHtml5Validate
       omitExtraData={omitExtraData ?? true}
-      showErrorList={false}
+      showErrorList={"top"}
```

Comment:
This is just so the reviewer can see the error messages, will remove before merging